snippets/database-next/read-and-write/rtdb_social_star_increment.js

```diff
@@ -0,0 +1,18 @@
+// This snippet file was generated by processing the source file:
```

Comment:
Sorry you have to run `npm run snippets` one more time!  It needs to be done every time there's a change to the vNext code.